deleteDeity.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+require 'src/db.php';
+
+$db = connectToDB('deitybase');
+$deities = getAllDeities($db);
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Deitybase</title>
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+</head>
+<body>
+    <main>
+        <form method="post" action="src/receiveDeleteDeityForm.php">
+            <?php foreach ($deities as $deity) {
+                echo "<p><input type='checkbox' name='" . $deity['id'] . "'value='on'>
+                <label>" . $deity['name'] . "</label></p>";
```

Comment:
label missing the for attribute and input missing the id attribute

Reply:
also, it is better to use css to arrange your inputs rather than a p tag

Reply:
or put it in a div instead of p

Reply:
also, the value attribute is not needed for the checkboxes because it is never used actually, see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox